src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import cn from 'classnames';
+
+type Props = {
+  onModalVisible: (arg: boolean) => void;
+  todo: Todo | null;
```

Comment:
```suggestion
  todo: Todo;
```
You don't need to render modal if you don't have todo

Reply:
Done